pkg/cmd/roachtest/tests/awsdms.go

```diff
@@ -68,13 +76,20 @@ const tableRules = `{
 	]
 }`
 
+var dmsEndpoints = make(map[string]string)
```

Comment:
can we do this without a global by e.g. keeping a `state` struct somewhere and passing it through everything?
also how do you feel about this being a struct instead of generic key value

Reply:
So the idea with the map was incase we have more sources and targets later on, it maps the name to its arn since to create the task it needs the endpoint arn. But when we create the endpoints, we don't know which tasks are going to use that endpoint so thats why I had like the key: "default_source". So the map is populated and then we look up the arn during the task creation. Can't quite see how I would utilize a struct the best here.

But yes, I can remove the global.

Reply:
having a static key is basically like having a struct :D so i'd imagine just

```
type dmsEndpointState struct {
   defaultSource string
   defaultTarget string
   bigBoiTarget string
}
```

but up to you.